src/agent/samples/policy/yaml/pod/pod-uvm-test-container.yaml

```diff
@@ -0,0 +1,21 @@
+spec:
+  runtimeClassName: kata-cc
+  containers:
+  - name: uvm-test
+    image: marinerconfpodstest.azurecr.io/kata-test-container:v1
+    imagePullPolicy: IfNotPresent
+    env:
+    - name: ENABLED_TESTS
+      value: "cpu,memory"
+    - name: TEST_CPU_EXPECTED_VCPU_COUNT
+      value: "4"
+    - name: TEST_MEMORY_EXPECTED_MEMORY_MB
```

Comment:
We internally also run this pod YAML against the vanilla handler where the observed memory values may diverge between kata and kata-cc - let's talk.